### PR TITLE
Set Atlas endpoint to ATLAS_ADDRESS if present

### PR DESCRIPTION
**Pull request by justincampbell**

### CODE REVIEW

File: v1/client.go [hunk 1]
```diff
@@ -60,6 +60,12 @@ type Client struct {
 
 // DefaultClient returns a client that connects to the Atlas API.
 func DefaultClient() *Client {
+	atlasEndpoint := os.Getenv("ATLAS_ADDRESS")
+
```
Comment (mitchellh) [inline v1/client.go]: Can you remove the newline here? We usually don't do that

File: v1/client_test.go [hunk 2]
```diff
@@ -10,8 +11,24 @@ import (
...
+}
+
+func TestDefaultClient_urlFromEnvVar(t *testing.T) {
+	otherEndpoint := "http://127.0.0.1:1234"
+
+	err := os.Setenv("ATLAS_ADDRESS", otherEndpoint)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.Unsetenv("ATLAS_ADDRESS")
```
Comment (mitchellh) [inline v1/client_test.go]: For maximum cleanness what you should do is `defer os.Setenv("ATLAS_ADDRESS", os.Getenv("ATLAS_ADDRESS"))` above all this.

File: v1/client.go [hunk 3]
```diff
@@ -60,6 +60,12 @@ type Client struct {
 
 // DefaultClient returns a client that connects to the Atlas API.
 func DefaultClient() *Client {
+	atlasEndpoint := os.Getenv("ATLAS_ADDRESS")
```
Comment (mitchellh) [inline v1/client.go]: Should probably make `ATLAS_ADDRESS` a constant, and I think we should rename it to `ATLAS_ADDR` to be more in line with our other projects (Consul, Vault, Nomad)

Comment (justincampbell) [inline v1/client.go]: Only chose `ATLAS_ADDRESS` because that's what Terraform uses. I didn't know about the env var in the others.

**Comment by sethvargo** [comment]
:+1: build is expected to fail because @cbednarski was gonna fix that archive bug :smile:

**Pull request merged**